settings.py

```diff
@@ -31,7 +31,8 @@
 # The site's URL (e.g. http://www.my-local-gym.com or http://localhost:8000)
 # This is needed for uploaded files and images (exercise images, etc.) to be
 # properly served.
-SITE_URL = 'https://wger-kronos.herokuapp.com/'
+# SITE_URL = 'https://wger-kronos.herokuapp.com/'
+SITE_URL = 'https://localhost/'
```

Comment:
@Emmanuel-Dominic could why did you switch back to localhost as `SITE_URL` does it affect the functionality of the application in any way?

Reply:
Sorry, @Rhytah,  that was when i was testing on my local machine with the development environment, however, thanks i have fixed that.